src/app/activity-panel/activity-panel.component.html

```diff
@@ -10,7 +10,10 @@ <h3 class="panelHeader">
       <tr *ngIf="(activity.unlocked)" draggable="true" (dragstart)="drag(activity, $event)">
         <td>
           <span class="noWrap">
-            <button (click)="onClick(activity)">{{activity.name[activity.level]}}</button>
+            <button (click)="onClick(activity, $event)" 
+                    tooltip="Add this activity to the bottom of your queue. Shift-click to start at 10 days.">
```

Comment:
Maybe "Shift click to also repeat it for 10 days."? And I think we're calling it a schedule rather than a queue most places where it's referenced, but double check that.

Reply:
Valid, lemme check and update this

Reply:
It says schedule in the button to remove items from the queue, so went ahead with that change